src/groovy/org/transmartproject/export/GENExporter.groovy

```diff
@@ -89,13 +92,13 @@ class GENExporter implements HighDimTabularResultExporter {
                 out << ' '
                 out << row.a2
                 
-                for (/*SnpLzCell*/ Object cell: row) { 
+                for (double[] cell: row) {
                     out << ' '
-                    out << cell.probabilityA1A1
+                    out << cell[0] // probabilityA1A1
                     out << ' '
-                    out << cell.probabilityA1A2
+                    out << cell[1] // probabilityA1A2
                     out << ' '
-                    out << cell.probabilityA2A2
+                    out << cell[2] // probabilityA2A2
```

Comment:
Yea, great, but I just changed the probabilities projection to return a Map instead of a double[] since the rest api only takes objects with properties. I think we need to decide if we want to fix the rest api implementation or just use maps.